package.json

```diff
@@ -16,6 +16,7 @@
       "buildutils",
       "buildutils/template",
       "buildutils/test-template",
+      "galata",
```

Comment:
This should be enough to have `@jupyterlab/galata` also released during the release process along with the other packages?

Reply:
Yup!